src/fides/api/models/privacy_experience.py

```diff
@@ -175,68 +175,38 @@ class PrivacyExperienceConfigHistory(ExperienceConfigBase, Base):
     )
 
 
-class PrivacyExperienceBase:
-    """Base Privacy Experience fields that are common between privacy experiences and historical records"""
-
-    disabled = Column(Boolean, nullable=False, default=False)
-    component = Column(EnumColumn(ComponentType), nullable=False)
-    region = Column(EnumColumn(PrivacyNoticeRegion), nullable=False, index=True)
-    version = Column(Float, nullable=False, default=1.0)
-
-    # Attribute that can be added as the result of "get_related_privacy_notices". Privacy notices aren't directly
-    # related to experiences.
-    privacy_notices: List[PrivacyNotice] = []
-
-    # Attribute that is cached on the PrivacyExperience object by "get_should_show_banner", calculated at runtime
-    show_banner: bool
-
-
-class PrivacyExperience(PrivacyExperienceBase, Base):
+class PrivacyExperience(Base):
     """Stores Privacy Experiences for a given just a single region.  The Experience describes how to surface
     multiple Privacy Notices to the end user in a given region.
 
     There can only be one component per region.
     """
 
+    component = Column(EnumColumn(ComponentType), nullable=False)
+    region = Column(EnumColumn(PrivacyNoticeRegion), nullable=False, index=True)
+
     experience_config_id = Column(
         String,
         ForeignKey(PrivacyExperienceConfig.id_field_path),
         nullable=True,
         index=True,
     )
 
-    experience_config_history_id = Column(
-        String,
-        ForeignKey(PrivacyExperienceConfigHistory.id_field_path),
-        nullable=True,
-        index=True,
-    )  # Also links to the historical record, so if the version of config gets updated, that
-    # triggers a new version of the experience.
-
     # There can only be one component per region
     __table_args__ = (UniqueConstraint("region", "component", name="region_component"),)
 
-    histories = relationship(
-        "PrivacyExperienceHistory", backref="privacy_experience", lazy="dynamic"
-    )
-
```

Comment:
I don't think there's any reason to track PrivacyExperienceHistory any longer, so I'm getting rid of this table and associated `relationship` properties.  Similarly there is no "version".  

"experience_config_history_id" was only here originally for versioning purposes, but the primary purpose of this PR, is that I'd already seen an example of where experience_config_id and experience_config_history_id were mismatched as I hadn't called the code to sync experiences to configs when the config was updated on startup.

Reply:
i like these simplifications a lot! they felt a bit overkill to me 😛